packages/checkbox-group/package.json

```diff
@@ -39,6 +39,9 @@
     "@lion/form-core": "0.6.14",
     "@lion/input": "0.10.12"
   },
+  "devDependencies": {
+    "@lion/validate-messages": "0.3.12"
```

Comment:
This may need to be a real dependency to ensure it is actually installed in extension layers that extend demos.